README.md

```diff
@@ -1,19 +1,19 @@
 # practices
 
-This document intends to document how we develop software at 2DII, including
+This document intends to document how we develop software at RMI, including
 social and technical aspects around a number of themes. It should help guide
 our actions,  and understand what to expect from our colleagues and what they
 expect from us.  Wherever possible, we refer to the source of information
 that defines a "best practice". Anecdotes and personal experience have
 value, but we prefer evidence from research, books, and large and mature
 communities of practice.
 
-We have agreed to this practices by consensus. They document heuristics, as
+We have agreed to these practices by consensus. They document heuristics, as
 detailed instructions may never cover all scenarios. We focus on _what_ to do
 and _why_, and only recommend optional resources about _how_ to do it.
 We realize there may be times when you can not achieve all of these practices.
 In those cases, it's nice to let others know, e.g. in a commit description or a
 PR comment, how or why some of these guidelines could not be met.
 
 * [Google Doc](https://bit.ly/2dii-practices)
```

Comment:
this is still 2dii specific

Reply:
I think this is ok, it's just the URL. The document is still useful (and if we end up making our own document, we can link it in the future)